contracts/src/StakeTable.sol

```diff
@@ -524,10 +549,43 @@ contract StakeTable is AbstractStakeTable {
         emit UpdatedConsensusKeys(msg.sender, node.blsVK, node.schnorrVK);
     }
 
-    /// @notice Minimum stake amount
-    /// @return Minimum stake amount
-    /// TODO: This value should be a variable modifiable by admin
-    function minStakeAmount() public pure returns (uint256) {
-        return 10 ether;
+    /// @notice Update the admin
+    /// @dev The admin cannot be set to the zero address
+    /// @param _admin The new admin
+    function updateAdmin(address _admin) external {
+        if (msg.sender != admin) revert Unauthorized();
```

Comment:
Is it a conscientious decision not to inherit from an existing contract with access control?

Reply:
you know what, i was thinking through it, wondered if this was more gas efficient

Reply:
switched to Ownable here https://github.com/EspressoSystems/espresso-sequencer/commit/dbcfeb3144509b493b104fd69fedb823d964cf16